app/templates/gulpfile.js

```diff
@@ -8,12 +8,13 @@ var buffer = require('buffer'),
     gulpJshint = require('gulp-jshint'),
     gulpUglify = require('gulp-uglify'),
     jshintStylish = require('jshint-stylish'),
-    map = require('map-stream');
+    map = require('map-stream'),
+    replace = require('gulp-replace');    
 
 gulp.task('scripts', function () {
   var header = new Buffer('// Copy this to your URL bar:\njavascript:');
 
-  gulp.src('app/{,*/}*.js')
+  return gulp.src('app/{,*/}*.js')
```

Comment:
mark task async in order to fulfill the dependency 

This is a nice write-up about it: http://blog.mais-h.eu/blog/2014/04/12/return-in-gulp-tasks/